### PR TITLE
Prospector Outfits & Locker Changes

### PR DESCRIPTION
**Pull request by Stutternov**
## About The Pull Request
<details>
<summary>
	About The Pull Request
</summary>
<hr>

Does as the title says.
- Re-adds the hunter crossbow to the Trapper locker as a spawn-chance weapon.
- Standardizes Prospector (now Shipbreaker), Salvager (now Trapper), and Foreman (now Field Shepherd) 'random' locker spawns.
- Gives Trappers the normal Hunter traits as before; pending some upcoming changes to it.
- Gives spawn armor and spawn gear upon coming up instead of relying off your locker to give you a random piece of armor.
- Re-sprites the Prospector shuttle
- Trades out random gear for proper loadouts; which are intended to be balanced decently.
- Disables half of the old Lodge crafting, makes the rest accessible to trappers unique crafting.

I wanted to do more in terms of adding a loadout system instead of the random locker spawns to prevent the constant player-habit of 'rerolling' all their loot gear constantly - but I just spent at least 4 hours on this due to how tedious making code for every since new uniform, armor, eyewear, face wear, name, description, and placing every sprite is. So - fuck it. This is what you get for now.
<hr>
</details>

## Changelog
:cl:
adds: Prospectors now get loadout options instead of random chance. Sidearms are still random chance though.
adds: New prospector gear
adds: New shuttle sprite
balance: Redoes random locker spawns; I do eventually want to change this into a loadout system (with random pistol spawns) - but this is how it's going to be for now.
removes: Bunch of redundant hunter weapons that don't serve a major use. Disables some of the others; makes the rest renamed to trappers.
/:cl:

### CODE REVIEW

File: code/modules/clothing/suits/fontaine.dm [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+		energy = 25,
+		bomb = 30,
+		bio = 0,
+		rad = 0
+	)
+	slowdown = 0.9
+
+/obj/item/clothing/suit/greatcoat/trapper
+	name = "Fontaine trapper greatcoat"
+	desc = "A Fontaine branded woolen coat made of fleece, hide, and wool of hunted or domesticated animals. It sports some light plating material between its layers, providing some basic protection. At least it's pretty warm though!"
```
Comment (WatermelonsEverywhere) [inline code/modules/clothing/suits/fontaine.dm:67]: Is the greatcoat supposed to have no armor? It doesn't seem like the base clothing/suit item has any, and there is no suit/greatcoat to inherit armor from.

File: code/game/objects/structures/crates_lockers/closets/secure/fontaine.dm [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+			new /obj/item/storage/backpack/duffelbag(src)
+		if("MILI")
+			new /obj/item/storage/backpack/military(src)
+
+	//longarms
+	switch(main_weapon_cache)
+		if("CROSSBOW")
+			new /obj/item/gun/projectile/shotgun/pump/hunter_crossbow(src)
+			new /obj/item/ammo_casing/crossbow_bolts(src)
+			new /obj/item/ammo_casing/crossbow_bolts(src)
```
Comment (WatermelonsEverywhere) [inline code/game/objects/structures/crates_lockers/closets/secure/fontaine.dm:162]: Unless this works differently than I expect it to, this spawns 2 singular crossbow bolts, if you'd rather spawn two stacks of 10, use crossbow_bolts/bulk

File: code/modules/clothing/suits/fontaine.dm (hunk 1 above)
Comment (Stutternov) [reply code/modules/clothing/suits/fontaine.dm:67]: Amended. It was supposed to call a parent of the general coat. Stupid way to set up greatcoat parents on the original code tbh.

File: code/game/objects/structures/crates_lockers/closets/secure/fontaine.dm (hunk 2 above)
Comment (Stutternov) [reply code/game/objects/structures/crates_lockers/closets/secure/fontaine.dm:162]: Amended; mis-read the 'max_ammount' as the 'spawn ammount'. Fixed.

**Pull request merged**